vpr/src/route/partition_tree.h

```diff
@@ -42,6 +55,8 @@ class PartitionTreeNode {
   public:
```

Comment:
TODO on line 53 is still outstanding?  If so, keep it (just checking).

Reply:
it was already done